### PR TITLE
DerClou! recipe - WIP

### PR DESCRIPTION
**Pull request by miqlas**
Compiles and runs from Terminal but not from Tracker, because it cannot find it files. Fix comes if i find a workaround.

### CODE REVIEW

File: games-engines/derclou/derclou-1.0.0~git.recipe [hunk 1]
```diff
@@ -0,0 +1,79 @@
+SUMMARY="A burglary simulation game"
+DESCRIPTION="You play Matt Stuvysant, a man of Polish descent and criminal persuasion. \
```
Comment (fbrosson) [inline games-engines/derclou/derclou-1.0.0~git.recipe]: The first line of DESCRIPTION exceeds the 80 char recommended limit.
For your convenience, here are the first lines with the "right" line breaks:
```
DESCRIPTION="You play Matt Stuvysant, a man of Polish descent and criminal \
persuasion. You arrive in London with nothing but the clothes on your back and \
a desire to become the greatest criminal mind this century. Once getting a \
room and meeting some accomplices, you must begin your career. Starting with \
small shops and gradually working your way up to the theft of the Crown Jewels.
```

File: games-engines/derclou/derclou-1.0.0~git.recipe [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	derclou$secondaryArchSuffix = $portVersion
+	app:derclou$secondaryArchSuffix = $portVersion
+	"
+
+REQUIRES="
```
Comment (fbrosson) [inline games-engines/derclou/derclou-1.0.0~git.recipe]: No need for a blank line between PROVIDES and REQUIRES.

File: games-engines/derclou/derclou-1.0.0~git.recipe [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+	lib:libSDL$secondaryArchSuffix
+	lib:libSDL_mixer$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libSDL$secondaryArchSuffix
+	devel:libSDL_mixer$secondaryArchSuffix
+	"
+
```
Comment (fbrosson) [inline games-engines/derclou/derclou-1.0.0~git.recipe]: No need for a blank line between BUILD_REQUIRES and BUILD_PREREQUIRES.

File: games-engines/derclou/derclou-1.0.0~git.recipe [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+HOMEPAGE="https://github.com/vcosta/derclou"
+COPYRIGHT="1994 neo Software Produktions GmbH
+	2000-2016 Vasco Alexandre da Silva Costa"
+LICENSE="DCPL"
+REVISION="1"
+SOURCE_URI="https://github.com/vcosta/derclou/archive/3d9fc7bfd27503d99c442f8a7564cefb32bb73fd.zip"
+SOURCE_URI_2="https://sourceforge.net/projects/cosp/files/Original%20Game%20Files/The%20Clue%21%20%2B%20Profidisk%20%28English%29/TheClueProfiData.zip"
+CHECKSUM_SHA256="40d1f2df686b9626bee5fdfb691b6937c718b69b9d006001ceed2c947c007f3a"
+CHECKSUM_SHA256_2="fe53142dc1c1ae2cd758e0eafa09ad4013027c50a15f88920e8bf2926d5279f3"
+SOURCE_DIR="derclou-3d9fc7bfd27503d99c442f8a7564cefb32bb73fd"
```
Comment (fbrosson) [inline games-engines/derclou/derclou-1.0.0~git.recipe]: I think you should replace SOURCE_URI by:
```
srcGitRev="3d9fc7bfd27503d99c442f8a7564cefb32bb73fd"
SOURCE_URI="https://github.com/vcosta/derclou/archive/$srcGitRev.zip"
```
and change SOURCE_DIR to:
```
SOURCE_DIR="derclou-$srcGitRev"
```

**Comment by waddlesplash** [comment]
> Fix comes if i find a workaround.

It looks like you added a patch workaround?

**Comment by miqlas** [comment]
Sadly it doesn't works. I should invert more time int ofixing old PR-s, but this game not really prio-1.

**Comment by waddlesplash** [comment]
(added tag on request of @miqlas)

**Comment by miqlas** [comment]
No interest anymore, closing for now.

**Pull request closed without merging**